### PR TITLE
VCR updates for tests in kubernetes repo

### PR DESCRIPTION
**Pull request by cben**
- No longer rely on hawkular metrics for example of volumes.
  Defined PVs, PVCs (successfully bound and pending), and referenced PVC in pod.
  Recorded without `--metrics` => minishift used ~2GB RAM instead of ~5GB :chart_with_downwards_trend: 

- Modify quotas between runs.
  https://bugzilla.redhat.com/show_bug.cgi?id=1504560

@miq-bot add-label test, gaprindashvili/yes

@zeari @yaacov @moolitayer please review (recommend by commits), together with https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/216

### CODE REVIEW

**Comment by cben** [comment]
@zeari @yaacov @moolitayer PTAL, this now has all I want.
Recommend reviewing by commits, the interesting ones are "Add volumes to VCR template, so we can test them in kubernetes repo" and "Quota changes for 1st and 2nd recording".

File: spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh [hunk 1]
```diff
@@ -63,8 +64,17 @@ describe_vcr () {
...
-  echo "== oc get images =="
-  oc get images
+  echo "== oc get resourceQuotas --all-namespaces --show-kind --show-labels =="
+  oc get resourceQuotas --all-namespaces --show-labels
+  echo
+  echo "== oc get persistentvolumes --show-kind --show-labels =="
+  oc get persistentVolumes --show-labels
+  echo
+  echo "== oc get persistentvolumeclaims --all-namespaces --show-kind --show-labels =="
+  oc get persistentVolumeClaims --all-namespaces --show-labels
```
Comment (moolitayer) [inline spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh]: We have a diff of `--show-kind` in the three echo. seems helpful to include, event if we are only showing one object kind at a time.

**Comment by cben** [comment]
Done, PTAL.
Will update k8s PR soon.

File: spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh [hunk 2]
```diff
@@ -96,12 +114,26 @@ oc label route my-route-2 key-route-label-
 oc patch --type=merge template my-template-2 --patch='{"parameters": []}'
 oc delete pod my-pod-2
 
+#old_quotas=$("oc describe quota
```
Comment (moolitayer) [inline spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh]: should this be here?

Comment (cben) [reply spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh]: thanks, removed

**Comment by cben** [comment]
this one good to go.

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit 75da91217e4c4a44f72a3297e65979ca1914bcac
Author: Mooli Tayer <mtayer@redhat.com>
Date:   Mon Jan 29 07:47:16 2018 +0200

    Merge pull request #83 from cben/vcr-for-kubernetes-too
    
    VCR updates for tests in kubernetes repo
    (cherry picked from commit 542fa531611ea6c100d7382f3b13698d64bd6034)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1559544
```

**Pull request merged**